interface/billing/sl_eob_search.php

```diff
@@ -212,7 +212,7 @@ function emailLogin($patient_id, $message)
         return true;
     } else {
         $email_status = $mail->ErrorInfo;
-        error_log("EMAIL ERROR: " . $email_status, 0);
+        error_log("EMAIL ERROR: " . errorLogEscape($email_status, 0));
```

Comment:
the second parameter of error_log should not be nested in errorLogEscape. should instead be:
```
error_log("EMAIL ERROR: " . errorLogEscape($email_status), 0);
```

Reply:
Got it!